.github/workflows/design-complete.yml

```diff
@@ -40,17 +40,25 @@ jobs:
 
               /* Modify labels */
 
-              // Remove "Design" label
-              await github.rest.issues.removeLabel({
-                ...issueProps,
-                name: "design"
-              });
+              // Try to remove "Design" label
+              try {
+                await github.rest.issues.removeLabel({
+                  ...issueProps,
+                  name: "design"
+                });
+              } catch(err) {
+                console.log(err);
```

Comment:
Can you add an additional message for context in addition to the caught error?

This also applies to the assigned label removal below.